src/components/views/dialogs/ReportEventDialog.tsx

```diff
@@ -42,6 +43,7 @@ interface IState {
     err?: string;
     // If we know it, the nature of the abuse, as specified by MSC3215.
     nature?: ExtendedNature;
+    ignoreUserToo: boolean; // if true, user will be ignored/blocked on submit
```

Comment:
not sure how I feel about the `too` - `ignoreUser` should be fine, no?

Reply:
it just didn't feel right to not acknowledge what the flag was doing. Given the context of the dialog, "ignore user" could mean "instead", "in addition to", or "don't file report". If we used the word "block" throughout the application instead, it'd be more than fine to leave it at `blockUser`.